Kinetic_shape_reconstruction/doc/Kinetic_shape_reconstruction/examples.txt

```diff
@@ -0,0 +1,6 @@
+/*!
+\example Kinetic_shape_reconstruction/kinetic_2d_example.cpp
```

Comment:
No "_example" suffix please.  Suffixes only in the test directory.

Reply:
example removed